model/project_ref_test.go

```diff
@@ -1847,6 +1847,25 @@ func TestGetProjectSetupCommands(t *testing.T) {
 	assert.Contains(t, cmds[1].String(), "c1")
 }
 
+func TestFindFirstProjectRef(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(ProjectRefCollection))
+
+	_, err := FindFirstProjectRef()
+	assert.Error(t, err, "Should return error and not panic if there are no matching projects")
```

Comment:
I think assert has `NotPanics` if we want to test that it doesn't panic. I think we don't usually concern ourselves to test for it explicitly, though, but idk why. Maybe because a panic would also abort the test and cause the test to return non zero?

Reply:
Yeah I assume that panicking automatically errors out the test. I can explicitly test that though.

Reply:
I added a NotPanics check... I mainly wanted to assert that it didn't panic because panicking would have been the result of me forgetting to do a bounds check on the resultant array, but that might just be kind of pointless?